### DISCUSSION TITLE
유니티로 월드 제작시 배경의 쉐이더는 어떤걸 사용해야 하나요?

### DISCUSSION THREAD

**Discussion by yoonjy51** [naverz/zepeto-studio-kor#520] [4. General]
유니티 내 standard 쉐이더를 사용해도 되는것인지..

캐릭터 의상 같은 경우는 custom shader 의 standard-js를 사용하는것 같더라구요.
배경 프랍들도 동일한 쉐이더 사용해도 되는건가요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 유니티에서 제공하는 쉐이더는 배경 제작시에도 사용 가능합니다. 
다만 렌더링 성능은 쉐이더에 직접적으로 영향을 받기 때문에 
사용시 성능 최적화 부분도 염두해 주시면 좋을 것 같습니다.
감사합니다.
